src/WebJobs.Extensions.DurableTask/Options/EmulatorStorageOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Options
+{
+    public class EmulatorStorageOptions : CommonStorageProviderOptions
+    {
+    }
```

Comment:
Add a coment inside the class definition explaining that there are no settings.

Reply:
Instead I added a remark in the xml that there is currently no configuration available for the Emulator storage provider. If you still think a comment inside the class definition is necessary, I will add one.